### PR TITLE
docs: C# client usage

### PR DESCRIPTION
**Pull request by Anush008**
This PR adds the C# client usage to the quick-start and concepts sections of the documentation.

[Preview](https://deploy-preview-599--condescending-goldwasser-91acf0.netlify.app/documentation/concepts/)

### CODE REVIEW

**Review by IvanPleshkov — APPROVED** [review]
Amazing job! I checked compilation of all snippets. But I didn't checked running result for each snippet, hope you did it. I leaved some not critical nitpicks

File: qdrant-landing/content/documentation/concepts/filtering.md [hunk 1]
```diff
@@ -628,6 +693,12 @@ import static io.qdrant.client.ConditionFactory.match;
 match("count", 0);
 ```
 
+```csharp
+using static Qdrant.Client.Grpc.Conditions;
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/filtering.md:697]: Will do.

File: qdrant-landing/content/documentation/concepts/collections.md [hunk 2]
```diff
@@ -1159,6 +1274,10 @@ client.create_alias("example_collection", "production_collection").await?;
 client.createAliasAsync("production_collection", "example_collection").get();
 ```
 
+```csharp
+await client.CreateAliasAsync("production_collection", "example_collection");
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/collections.md:1278]: Will do for all the snippets.

File: qdrant-landing/content/documentation/quick-start.md [hunk 3]
```diff
@@ -482,6 +590,20 @@ List<ScoredPoint> searchResult =
 System.out.println(searchResult);
 ```
 
+```csharp
+using static Qdrant.Client.Grpc.Conditions;
+
+var searchResult = await client.SearchAsync(
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/quick-start.md:596]: Here, `client` is defined in the snippet above. Similar the other language snippets.

File: qdrant-landing/content/documentation/concepts/filtering.md [hunk 4]
```diff
@@ -230,12 +242,24 @@ client
...
 
+```csharp
+using Qdrant.Client;
+using static Qdrant.Client.Grpc.Conditions;
+
+var client = new QdrantClient("localhost", 6334);
+
+await client.ScrollAsync(
+	"{collection_name}",
+	MatchKeyword("city", "London") | MatchKeyword("color", "red")
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/filtering.md]: https://github.com/qdrant/landing_page/pull/599/commits/f1ba92770a8ce4f4523a2ccc2ebce40ab199beae

File: qdrant-landing/content/documentation/concepts/filtering.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/filtering.md:702]: https://github.com/qdrant/landing_page/pull/599/commits/3c4f79f746e4e3ac122e0ee5d6ad206f0820c80b

File: qdrant-landing/content/documentation/concepts/collections.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/concepts/collections.md]: https://github.com/qdrant/landing_page/pull/599/commits/3c4f79f746e4e3ac122e0ee5d6ad206f0820c80b

**Pull request merged**